web/src/components/profile/approved-researcher/TrainingCertificate.tsx

```diff
@@ -0,0 +1,101 @@
+import Button from "@/components/ui/Button";
+import LoginFallback from "@/components/ui/LoginFallback";
+import { useAuth } from "@/hooks/useAuth";
+import { postProfileTraining } from "@/openapi";
+import { useState } from "react";
+import styles from "./TrainingCertificate.module.css";
+import TrainingCertificateError from "./TrainingCertificateError";
+
+interface FormEvent extends React.FormEvent<HTMLFormElement> {
+  target: HTMLFormElement;
+}
+
+export default function TrainingCertificate() {
+  const { loading, isAuthed } = useAuth();
+  const [isValid, setIsValid] = useState<boolean | undefined>(undefined);
+  const [errorMessage, setErrorMessage] = useState<string | undefined>(undefined);
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    const { certificate } = e.target;
+    const files: FileList = certificate.files;
+    if (files.length != 1) {
```

Comment:
shouldn't actually be possible since we haven't added [`multiple`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/file#multiple)

Reply:
fine to keep it to be doubly sure 👍